### ISSUE TITLE
<Foundations-Flexbox>: <new additional resource: easy explanation>

### ISSUE THREAD

**Issue by Confused-code** [TheOdinProject/curriculum#28396]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

So i was reading through an article by internet is hard on [flexbox](https://internetingishard.netlify.app/html-and-css/flexbox/index.html) (as additional resource), and some of the explaination felt incomplete, specially when i read order in flexbox model, it was so confusing.

after a google search i reached a site named SamanthaMing.com's article on order which contained only single image, see [here](https://www.samanthaming.com/flexbox30/20-order/), it felt like self realization at spiritual level, with just one look at it.

My suggestion is if this resource makes sense to you guys and the pwner agrees, may be it can be introduced to others as additional resource.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-introduction-to-flexbox

**Comment by JoshDevHub** [comment]
Thank you for raising this issue @Confused-code 

Going to check with the @TheOdinProject/foundations team to see what their opinions are on this resource.

**Comment by thatblindgeye** [comment]
Personally I have 2 viewpoints:

1. It almost feels like this resource is *too* simple. It covers an extremely narrow/specific facet of flex, so I'm not entirely sure the importance of offering it as an additional resource
2. I feel we shouldn't promote fiddling with things like the `order` property for people most likely just starting out, at least with this sort of context where it's just stating the CSS property/showing how to use it without really going into more details surrounding it. Doing so can have pretty big, negative side effects that people should be aware of (namely that while visually you're updating the `order` of flex items, the tab order won't match up and can be confusing at best)

**Comment by KevinMulhern** [comment]
I agree, IIRC we don't cover flex order in any of basic flexbox lessons and shouldn't link to any resources that cover it specifically. We could remove the interneting is hard tutorial to later on, or remove it?

**Comment by CouchofTomato** [comment]
Do we have a consensus on any action needed here?

**Comment by ManonLef** [comment]
We decided we have all the information needed on flex box already covered in the lesson so we close this
